sql/item_weapon.sql

```diff
@@ -3709,7 +3714,7 @@ INSERT INTO `item_weapon` VALUES (20756,'soulcleaver',4,0,228,228,188,2,1,489,28
 INSERT INTO `item_weapon` VALUES (20757,'foreshock_sword',4,0,242,242,188,2,1,504,273,0);
 INSERT INTO `item_weapon` VALUES (20758,'mekosuchus_blade',4,0,228,228,188,2,1,504,273,0);
 INSERT INTO `item_weapon` VALUES (20759,'macbain',4,0,242,242,188,2,1,510,268,0);
-INSERT INTO `item_weapon` VALUES (20760,'home._claymore',4,0,215,215,167,2,1,444,228,0);
+INSERT INTO `item_weapon` VALUES (20760,'home._claymore',4,0,215,215,167,2,1,444,240,0);
```

Comment:
ilvl_skill 242
ilvl_parry 242
ilvl_macc 188

Reply:
will fix but i didnt set any ilv stats, whatever i used to update this list didn't include those